python/py-cattrs/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-cattrs
+version             1.8.0
+revision            0
+license             MIT
+maintainers         nomaintainer
+supported_archs     noarch
+
+description         cattrs is an open source Python library for structuring and unstructuring data.
+long_description    {*}${description}. cattrs works best with attrs classes, dataclasses and the usual Python collections, \
+                    but other kinds of classes are supported by manually registering converters.
+
+homepage            https://github.com/python-attrs/cattrs
+
+python.versions     37 38 39 310
+
+checksums           rmd160  7cc352191d51c4cfbedc1eacf8ad5b61045736ca \
+                    sha256  5c121ab06a7cac494813c228721a7feb5a6423b17316eeaebf13f5a03e5b0d53 \
+                    size    22501
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-attrs \
```

Comment:
in this case it's probably harmless, but it would still be best not to have a stray backslash at the end of this line

Reply:
good catch - fixed